frontend/src/js/components/header/header.js

```diff
@@ -142,6 +145,17 @@ const useStyles = makeStyles()(theme => ({
   organization: { marginBottom: theme.spacing() },
   redAnnouncementIcon: {
     color: theme.palette.error.dark
+  },
+  spTenantBadge: {
+    padding: theme.spacing(1),
+    color: theme.palette.grey[800],
+    background: theme.palette.tooltip.tierTipBackground,
+    borderRadius: theme.spacing(3),
+    marginRight: theme.spacing(2),
+    fontSize: '11px'
+  },
+  spTenantBadgeContainer: {
+    borderRight: `1px solid ${theme.palette.grey[300]}`
   }
```

Comment:
there is the `.header-section` class that handles this for the existing widgets in the header - I guess that would be a good reason to keep the header related content in a less file in the header folder to make it more apparent there is some related stuff to be found